tests/test/simpleStorage.js

```diff
@@ -1,11 +1,10 @@
 const { execSync } = require('child_process');
-const { assert } = require('console');
 const SimpleStorage = artifacts.require("SimpleStorage");
 const Web3 = require('web3');
 
 const source_dsl = 'http://localhost:8545';
 const target_dsl = 'http://localhost:8540';
-const target_address = '0x00ce0c25d2a45e2f22d4416606d928b8c088f8db';
+const continueMigrationFile = 'test/data/interruptedMigration.json';
```

Comment:
the old one is deprecated. I only set a continueMigrationFile so that the tests don't interfere with the normal workflow. I imagined the scenario where a user would try to migrate a contract, it fails and the output is written to interruptedMigration.json. The user changes the code and runs the tests for verification. Then, the interruptedMigration.json would be overwritten if I wouldn't have added this custom path for the interruptedMigration.